tox.ini

```diff
@@ -17,9 +17,9 @@ passenv = TEST_MYSQL_* TEST_POSTGRESQL_*
 basepython = python2.7
 changedir = webapp
 commands =
-	coverage run --branch manage.py test
-	coverage xml --include=graphite/*
-	coverage report --include=graphite/*
+	coverage run --branch --include=graphite/* manage.py test
+	coverage xml
+	coverage report
 deps =
```

Comment:
This is just an improvement to the efficiency of the tests, because coverage isn't going to collect info on files outside of graphite only to discard it when generating the reports.